### PR TITLE
fix:  ListComments order of gitlab

### PR DESCRIPTION
**Pull request by wowq**
Fix the issue of listcomments reading order in gitlab so that the latest variables can be read

### CODE REVIEW

File: pkg/cmd/pr/variables/variables.go [hunk 1]
```diff
@@ -179,7 +179,9 @@ func (o *Options) modifyVariables(text string) error {
 func (o *Options) loadPRComments(envVars map[string]string, pr *scm.PullRequest) error {
 	ctx := o.GetContext()
 	prNumber := pr.Number
-	opts := scm.ListOptions{}
+	opts := scm.ListOptions{
```
Comment (ankitm123) [inline pkg/cmd/pr/variables/variables.go:182]: Is there any chance this will break for other scm providers?

Comment (wowq) [reply pkg/cmd/pr/variables/variables.go:182]: No, now only gitlab provider handles this parameter. ASC is the default in GitHub

**Pull request merged**